truckersmp-cli

```diff
@@ -713,7 +713,7 @@ def check_args_errors():
         if args.proton and args.wine:
             sys.exit("Start with Proton (-p) or Wine (-w)?")
         elif not args.proton and not args.wine:
-            sys.exit("Only possible to start with Proton (-p) or Wine (-w).")
+            args.proton = True
```

Comment:
Proton is available only on Linux systems.
I would like to add `logging.info()` (or `logging.debug()`) too.
```suggestion
            if platform.system() == "Linux":
                logging.info("Platform is Linux, use Proton")
                args.proton = True
            else:
                logging.info("Platform is not Linux, use Wine")
                args.wine = True
```